nextflow.config

```diff
@@ -208,7 +208,7 @@ try {
 
 profiles {
     debug {
-        dumpHashes             = true
+        dumpHashes             = false
```

Comment:
Was this change intended? If yes I'm not sure if this general debugging option should be changed hidden away in another topics PR!

Reply:
No it shouldn't be changed,it's from template... I had also left a comment in my review on this but apparently didnt' save(?!)